spec/models/user_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User do
+  let(:invalid_email) do
+    described_class.create(email: 'invalid',
```

Comment:
You can use
```suggestion
    build(:user, email: 'invalid',
```

Reply:
Simplified creation user with invalid email.